kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -885,6 +926,114 @@ public void handleSendAndRecordButtonView(boolean isSendButtonVisible) {
         recordButton.setVisibility(showRecordButton ? (isSendButtonVisible ? View.GONE : View.VISIBLE) : View.GONE);
     }
 
+    @Override
+    public void onMessageSent(Message message) {
```

Comment:
Why are all these empty?

Reply:
The ApplozucUIListener came later. Other callbacks that the ApplozicUIListener are being done in others ways.
For the feedback feature, The onChannelUpdate of the ApplozicUIListener was the best to implement. 
The rest arent required, so are empty.

Reply:
ApplozicUIListener (the interface that has these functions) is a new addition. Here we require only the channel update function. The rest functions have been implemented in other ways.